neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/common/impl/RemoteNeo4jConfiguration.java

```diff
@@ -24,6 +24,11 @@
 
 	public static final int DEFAULT_BOLT_PORT = 7687;
 
+	/**
+	 * Default Size of the client connection pool used by the RestEasy HTTP client
+	 */
+	public static final int DEFAULT_REST_CONNECTION_POOL_SIZE = 10;
```

Comment:
Please, rename this to DEFAULT_CONNECTION_POOL_SIZE

Reply:
Hi @DavideD !
Ok. 5 minutes